packages/jsii-pacmak/lib/targets/go/types/enum.ts

```diff
@@ -11,6 +12,10 @@ export class Enum extends GoType {
 
   public constructor(pkg: Package, public type: EnumType) {
     super(pkg, type);
+
+    this.stdImports.add('encoding/json');
+    this.stdImports.add('fmt');
+    this.stdImports.add('errors');
```

Comment:
I'd rather we follow the Law of Demeter here... Actually, since we're in the constructor, and this is the only use of `this.stdImports` (from `GoType`) being accessed yet, it would make sense that the values are provided to the `super` constructor.

This would allow `GoType` to (publicly) expose only a `ReadonlySet`.

Reply:
Sure, can do

Reply:
Done